docs/user/alerting/alerting-troubleshooting.asciidoc

```diff
@@ -1,238 +1,200 @@
-              "sum_other_doc_count" : 0,
-              "buckets" : [
-                {
-                  "key" : "41893910-6bca-11eb-9e0d-85d233e3ee35",
-                  "doc_count" : 6
-                }
-              ]
-            }
-          }
-        }
-      ]
-    }
+    "task" : {
+      "schedule" : {
+        "interval" : "5s"
+      },
+      "taskType" : "alerting:.index-threshold",
+      "retryAt" : null,
+      "runAt" : "2021-05-10T05:18:02.704Z",
+      "scope" : [
+        "alerting"
+      ],
+      "startedAt" : null,
+      "state" : """{"alertInstances":{},"previousStartedAt":"2021-05-10T05:17:45.671Z"}""",
+      "params" : """{"alertId":"30d856c0-b14b-11eb-9a7c-9df284da9f99","spaceId":"default"}""",
+      "ownerId" : null,
+      "scheduledAt" : "2021-05-10T04:50:07.333Z",
+      "attempts" : 0,
+      "status" : "idle"
+    },
+    "references" : [ ],
+    "updated_at" : "2021-05-10T05:17:58.000Z",
+    "coreMigrationVersion" : "8.0.0",
+    "type" : "task"
   }
 }
 --------------------------------------------------
-<1> Most rule execution durations fall within the first bucket (0 - 1 seconds).
-<2> A single rule with id `41893910-6bca-11eb-9e0d-85d233e3ee35` took between 30 and 31 seconds to execute.
 
-Use the <<get-rule-api,Get Rule API>> to retrieve additional information about rules that take a long time to execute.
\ No newline at end of file
+What you can see above is the task that backs the rule, and for the rule to work, this task must be in a healthy state. This information is available via <<task-manager-api-health, health API>> or via verbose logs if debug logging is enabled.
+When diagnosing the health state of the task, you will most likely be interested in the following fields:
+
+`status`:: This is the current status of the task.  Is Task Manager is currently running? Is Task Manager idle,  and you’re waiting for it to run?  Or has Task Manager has tried to run it and failed?
+`runAt`:: This is when the task is scheduled to run next. If this is in the past and the status is idle, Task Manager has fallen behind or isn’t running. If it’s in the past, but the status is running, then Task Manager has picked it up and is working on it, which is considered healthy.
+`retryAt`:: Another time field, like runAt. If this field is populated, then Task Manager is currently running the task. If the task doesn’t complete (and isn't marked as failed), then Task Manager will give it another attempt at the time specified under retryAt.
+
+Investigating the underlying task can help you gauge whether the problem you’re seeing is rooted in the alert not running at all, whether it’s running and failing, or whether it is running, but exhibiting behavior that is different than what was expected (at which point you should focus on the rule itself, rather than the task).
+
+In addition to the above methods, broadly used the next approaches and common issues:
+
+* <<alerting-common-issues, Alerting common issues>>
```

Comment:
These are not listed in the same order as they show up in the table of contents. Can we have them in the same order?